src/main/java/org/broadinstitute/hellbender/tools/funcotator/vcfOutput/VcfOutputRenderer.java

```diff
@@ -75,44 +75,44 @@
     /** List of the fields that will get rendered in the funcotation annotation.  Excluded fields have been removed.  */
     private final List<String> finalFuncotationFieldNames;
 
-    //==================================================================================================================
-
-    public VcfOutputRenderer(final VariantContextWriter vcfWriter,
-                             final VCFHeader existingHeader,
-                             final List<DataSourceFuncotationFactory> dataSources) {
-        this(vcfWriter, dataSources, existingHeader, new LinkedHashMap<>(), new LinkedHashMap<>());
-    }
-
-    public VcfOutputRenderer(final VariantContextWriter vcfWriter,
-                             final List<DataSourceFuncotationFactory> dataSources,
-                             final VCFHeader existingHeader,
-                             final LinkedHashMap<String, String> unaccountedForDefaultAnnotations,
-                             final LinkedHashMap<String, String> unaccountedForOverrideAnnotations) {
-        this(vcfWriter, dataSources, existingHeader, unaccountedForDefaultAnnotations, unaccountedForOverrideAnnotations,
-                new LinkedHashSet<>(), new LinkedHashSet<>());
-    }
-
-    @VisibleForTesting
-    public VcfOutputRenderer(final VariantContextWriter vcfWriter,
-                             final List<DataSourceFuncotationFactory> dataSources,
-                             final VCFHeader existingHeader,
-                             final LinkedHashMap<String, String> unaccountedForDefaultAnnotations,
-                             final LinkedHashMap<String, String> unaccountedForOverrideAnnotations,
-                             final Set<VCFHeaderLine> defaultToolVcfHeaderLines) {
-        this(vcfWriter, dataSources, existingHeader, unaccountedForDefaultAnnotations, unaccountedForOverrideAnnotations,
-               defaultToolVcfHeaderLines, new HashSet<>());
-    }
+    /** The version of the tool used to produce the VCF file. */
+    private final String toolVersion;
```

Comment:
Might as well put this as a field in the `OutputRenderer` parent class itself.  I can't imagine ever not needing it.

Reply:
Done